new_core_concepts/modules/ROOT/pages/typeql/entities-relations-attributes.adoc

```diff
@@ -1,4 +1,137 @@
 = Entities, Relations, Attributes
+// TODO: Out links to manual / fundamentals articles
 
-[placeholder]
-TypeQL data model concepts. 
\ No newline at end of file
+TypeDB implements the Polymorphic Entity-Relation-Attribute (PERA) data model.
+A TypeDB schema defines a hierarchy of these types, and how they interact with each other through interface-types.
+
+=== Entity
+
+Entities are standalone types, which exist independent of other types.
+
+E.g. A `person` exists, regardless of whether they are involved in friendships or have a name.
+[,typeql]
+----
+entity person;
+----
+
+=== Relation
+Relations are types with one-or-more associated "role" types.
+An instance of a relation "depends" on instances of other types which "play" these roles.
+
+E.g. An `employment` relation exists to relate an `employer` to an `employee`.
+An `employment` instance cannot exist without the other instances which play these roles.
+[,typeql]
+----
+relation employment, relates employer, relates employee;
+----
+
+=== Attribute
+Attributes are types which hold a value.
+An instance of an attribute type is identified by its value.
+
+E.g. An `age` 10.
+
+An attribute type must specify the primitive `value-type` of the values it will hold
+
+[,typeql]
+----
+attribute age, value integer;`
+----
+
+== Owning attributes & playing roles
+
+Entity & relation types may be defined to "own" attribute types.
+
+E.g. A `person` may own an attribute `birth-date`, or an `employment` may own a `start-date`;
+[,typeql]
+----
+person owns age;
+----
+
+Entity & relation types may be defined to "play" a role defined by a relation.
+
+E.g. The entity type `company` may play the  `employer` role of an `employment` relation.
+
+[,typeql]
+----
+company plays employment:employer;
+----
+
+== Polymorphism
+
+Polymorphism in TypeDB comes is realised in two ways:
+
+=== Interfaces
+Multiple types may "implement" the same interface. In the following schema snippet, the `employer` role in the `employment` relation can be played by a `company` or an `charity`.
```

Comment:
I think it's worth pulling up the idea that a `role` is a kind of interface you define by creating the role, and can impement with `plays`, and that attributes define an implicit interface you impement with `owns`.

Then go on to say that:

< multiple types may implement the same interface (i.e. use own the same attribute, or play the same role >

Again this is a new/different idea so I think worth repeating multiple times in slightly different ways.